packages/extension/src/pages/main/account.tsx

```diff
@@ -49,25 +50,43 @@ export const AccountView: FunctionComponent = observer(() => {
               intl.formatMessage({
                 id: "setting.keyring.unnamed-account",
               })
+            : accountInfo.walletStatus === WalletStatus.Rejected
+            ? "Unsupported Key"
             : "Loading..."}
         </div>
         <div style={{ flex: 1 }} />
       </div>
-      <div className={styleAccount.containerAccount}>
-        <div style={{ flex: 1 }} />
-        <div
-          className={styleAccount.address}
-          onClick={() => copyAddress(accountInfo.bech32Address)}
+      {accountInfo.walletStatus === WalletStatus.Rejected && (
```

Comment:
Just because a wallet status is rejected does not guarantee that the error comes from loading pub keys